manifests/otel-agent-cm.yaml

```diff
@@ -32,6 +32,29 @@ data:
         tls:
           insecure: true
     processors:
+      attributes:
```

Comment:
Is this adding resource attributes or metric labels?

Reply:
The processor modifies attributes: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor .

Not sure that is what we want. I thought the plan is to convert these attributes into metric labels.

@tiffanny29631 , let us get your design doc approved first before Karl and I spent more reviewing this PR.

Reply:
The design doc is ready for another look go/cs-resource-to-metric-label. 

Per discussion in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/534, the attributes added by attribute processor are metrics labels.